pep/app/pods/most-cited/controller.ts

```diff
@@ -11,6 +11,10 @@ import IntlService from 'ember-intl/services/intl';
 import Journal from 'pep/pods/journal/model';
 import { PERIODS, PossiblePeriodValues } from 'pep/constants/sidebar';
 import { QueryParams } from 'pep/hooks/useQueryParams';
+import { buildQueryParams } from '@gavant/ember-pagination/utils/query-params';
+import { urlForDocumentQuery } from 'pep/pods/document/adapter';
```

Comment:
@bakerac4  doesnt look like u are using `urlForDocumentQuery` in this file either